auth/auth_test.go

```diff
@@ -9,6 +9,9 @@ import (
 )
 
 func TestMain(m *testing.M) {
+	if url := os.Getenv("TEST_DATABASE_URL"); url == "" {
+		return
+	}
```

Comment:
This is 100% wrong. I'm not sure what I was thinking here, but this bails out of all tests in this package if the database isn't set which is...not what I wanted. I'll need to come up with a different way to run DB integration tests selectively.